omod/src/test/java/org/openmrs/module/webservices/rest/doc/SwaggerSpecificationCreatorTest.java

```diff
@@ -125,16 +126,47 @@ public void checkRepresentationParamExists() throws Exception {
 			}
 		}
 	}
-
+	
 	private boolean operationHasRepresentationParam(Operation o) {
 		boolean ret = false;
-
+		
 		for (Parameter p : o.getParameters()) {
 			if (p.getName().equals("v")) {
-				ret = true;
+				ret = !ret;
 			}
 		}
-
+		
 		return ret;
 	}
+	
+	// make sure each operation that supports paging has the limit and startIndex parameters
+	@Test
+	public void checkPagingParamsExist() throws Exception {
+		SwaggerSpecificationCreator ssc = new SwaggerSpecificationCreator("/v1/");
+		ssc.BuildJSON();
+		SwaggerSpecification spec = ssc.getSwaggerSpecification();
+		
+		for (Path p : spec.getPaths().getPaths().values()) {
+			for (Operation o : p.getOperations().values()) {
+				if (o.getOperationId().matches("^getAll[A-Z].*")) {
+					Assert.assertTrue("Ensure each operation that supports paging has both paging parameters",
+					    operationHasPagingParams(o));
+				}
+			}
+		}
+	}
+	
+	private boolean operationHasPagingParams(Operation o) {
+		boolean limit = false, startIndex = false;
+		
+		for (Parameter p : o.getParameters()) {
+			if (p.getName().equals("limit")) {
```

Comment:
No big deal. Though i feel it would be clearer and more obvious if you just assigned true. :)